drivers/OCFSSR.py

```diff
@@ -120,7 +120,7 @@ def detach(self, sr_uuid):
         except util.CommandException, inst:
                 raise xs_errors.XenError('OCFSUnMount', \
                       opterr='Failed to umount FS. Errno is %d' % \
-                      os.strerror(inst.code))
+                      inst.code)
```

Comment:
If these aren't actually errnos then the message string is also incorrect. Replace Errno with Error is probably best.